### PR TITLE
feat: added a new checkAddressesMine method to both facades

### PR DESCRIPTION
**Pull request by andreabadesso**
### Acceptance Criteria
- Include here all things that this PR should solve

### Security Checklist
- [X] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

**Review by tuliomir — APPROVED** [review]

File: src/wallet/types.ts [hunk 1]
```diff
@@ -81,6 +81,11 @@ export interface AddressesResponseData {
   addresses: GetAddressesObject[];
 }
 
+export interface CheckAddressesMineResponseData {
+  success: boolean;
+  addresses: WalletAddressMap;
```
[the remark this answers is not in the document]
Comment (andreabadesso) [reply src/wallet/types.ts]: It is, fixed in d0a9e75e74ca2739a59781da9d0e9b6de934ce29

File: src/new/wallet.js [hunk 2]
```diff
@@ -2173,6 +2173,21 @@ class HathorWallet extends EventEmitter {
     return wallet.isAddressMine(address);
   }
 
+  /**
+   * Check if a list of addresses are from the loaded wallet
+   *
+   * @param {object} address Address to check
+   *
+   * @return {Object} Object with the addresses and whether it belongs of not { address: boolean }
```
[the remark this answers is not in the document]
Comment (andreabadesso) [reply src/new/wallet.js]: Fixed in d0a9e75e74ca2739a59781da9d0e9b6de934ce29

File: src/new/wallet.js [hunk 3]
```diff
@@ -2173,6 +2173,21 @@ class HathorWallet extends EventEmitter {
     return wallet.isAddressMine(address);
   }
 
+  /**
+   * Check if a list of addresses are from the loaded wallet
+   *
+   * @param {object} address Address to check
```
[the remark this answers is not in the document]
Comment (andreabadesso) [reply src/new/wallet.js]: Fixed in d0a9e75e74ca2739a59781da9d0e9b6de934ce29

File: src/wallet/types.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (andreabadesso) [reply src/wallet/types.ts:86]: It is never optional, when success is false we throw an error

**Review by pedroferreira1 — APPROVED** [review]

**Pull request merged**